src/styles/mixins.scss

```diff
@@ -0,0 +1,12 @@
+@mixin location($translateX, $translateY, $rotate) {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(#{$translateX}, #{$translateY}) rotate(#{$rotate});
```

Comment:
```suggestion
  transform: translate(#{$translateX}, #{$translateY}) rotate(#{$rotate});
  transform-origin: bottom
```

Reply:
why should i use "bottom" for transform-origin?